README.md

```diff
@@ -109,11 +109,11 @@ What makes Matomo unique from the competition:
   * Modern, easy to use User Interface: you can fully customize your dashboard, drag and drop widgets and more.
 
   * Matomo features are built inside plugins: you can add new features and remove the ones you don’t need.
-    You can build your own web analytics plugins or hire a consultant to have your custom feature built in Matomo
+    You can build your own web analytics plugins or hire a consultant to have your custom feature built-in Matomo
 
-  * Vibrant international Open community of more than 200,000 active users (tracking even more websites!)
+  * A vibrant international Open community of more than 200,000 active users (tracking even more websites!)
 
-  * Advanced Web Analytics capabilities such as Ecommerce Tracking, Goal tracking, Campaign tracking,
-    Custom Variables, Email Reports, Custom Segment Editor, Geo Location, Real time maps, and more!
+  * Advanced Web Analytics capabilities such as E-commerce Tracking, Goal tracking, Campaign tracking,
```

Comment:
FYI: I think in all docs Matomo uses "Ecommerce" over "E-commerce". Not sure why though (wanted to change it myself some time as well), the only non-hyphenated alternative should be eCommerce.

Reply:
cc @mattab